mne/epochs.py

```diff
@@ -842,8 +842,19 @@ def plot_psd(self, fmin=0, fmax=np.inf, proj=False, n_fft=256,
             End frequency to consider.
         proj : bool
             Apply projection.
-        n_fft : int
-            Number of points to use in Welch FFT calculations.
```

Comment:
why? did I miss some discussion? why is it not deprecated if it has to be removed?